src/sage/schemes/projective/projective_subscheme.py

```diff
@@ -1515,3 +1515,77 @@ def local_height_arch(self, i, prec=None):
             4.61512051684126
         """
         return self.Chow_form().local_height_arch(i, prec)
+
+    def coherent_sheaf(self, module, twist=0):
+        r"""
+        Return the sheaf defined by the graded ``module``.
+
+        If ``twist`` is a non-zero integer `n`, the sheaf twisted by
+        `\OO_{\PP^r}(n)` is returned.
+
+        INPUT:
+
+        - ``module`` -- a free module or a quotient module
+
+        - ``twist`` -- (default: `0`) an integer
```

Comment:
```suggestion
        - ``twist`` -- (default: ``0``) an integer
```
since it is code input :)

Reply:
OK.